R/build-reference.R

```diff
@@ -251,6 +251,25 @@ build_reference_topic <- function(topic,
     }
   )
 
+  deps <- data$dependencies
+  if (data$has_deps <- !is.null(deps)) {
```

Comment:
Do you mind doing the assignment outside of the if statement? Generally, we only do assignments at the top-level.

Reply:
Done now.